packages/access-api/test/access-authorize.test.js

```diff
@@ -119,7 +119,7 @@ describe('access/authorize', function () {
 
     const url = new URL(inv)
     // click email url
-    await mf.dispatchFetch(url)
+    await mf.dispatchFetch(url, { method: 'POST' })
```

Comment:
Why does the comment say 'click email url', which I assume would usually involve an HTTP GET, but this changes it to 'POST'?

Reply:
right so this change splits the approval process into 3 pieces (from the 2 that it is now):
1) an email with a link 
2) a page that gets served on GET that uses JavaScript to redirect users to
3) a page that gets served on POST that does the actual verification and triggers the websocket data flow

here's the page that gets served by the GET request:

https://github.com/web3-storage/w3protocol/pull/430/files#diff-1a9cb8eda7a1020a96a7a7c0db5406ee04e5f8953fb6c00ab35f3bb8e6fb2519R103

Reply:
and for clarity, in this framing our current process is:

1) an email with a link
2) a page that gets served on GET that does the verification and triggers the websocket data flow